src/components/pages/alerts/Alert.jsx

```diff
@@ -10,7 +10,14 @@ import ExampleAlert from '../../../assets/uploaded_blueprints/alerts-floor-plan-
 
 const useStyles = makeStyles({
    alertCards: {
-      marginTop: '3vh'
+      marginTop: '3vh',
+
+      '&:hover': {
+         transition: '0.13s',
+         backgroundColor: '#F4FFFF',
+         borderColor:'#A2DAD9',
+         borderWidth: 2.5,
```

Comment:
Look good! Could you find a way to make the hover effect not shrink the card? I think it's the border when you hover over it. It tries to keep the same dimensions. You could either enlarge the card itself on hover or have a white border when it's not being hovered over (lol).

Reply:
Hey Shandy, I believe it's fixed:)